docs/FAQ.md

```diff
@@ -8,6 +8,12 @@ this document. Few of answers assumes that the MCM is being used in conjuction w
 <!--- TOC BEGIN -->
 * [Basics](#basics)
   * [What is Machine Controller Manager?](#what-is-machine-controller-manager)
+  * [What are different sub-controllers in MCM ?](#What-are-different-sub-controllers-in-MCM)
+  * [What is safety-controller in MCM ?](#What-is-safety-controller-in-MCM)
+  * [Why is my machine deleted ?](#Why-is-my-machine-deleted)
```

Comment:
Moved this FAQ to the end of basics section so as to group the informative and technical questions in the beginning and the end of the section.